### ISSUE TITLE
[CTA Section]: QA: Text elements in the feature are getting truncated when text spacing is applied

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#9106]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2526
- DPO consulting engineer: N/A

-----

### Description

Page element:
Text elements and links

Issue : Elements are getting overlapped with one another on applying text spacing.

Expected: On applying text spacing, page should get appended accordingly. No overlap or truncation should happen.

User Impact: If there is a loss of content or functionality When users override author specified text spacing to improve their reading experience, it affects users with low vision / user suffering from dyslexia

### Component(s) impacted

CTA-Section

### Browser

Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by oliviaflory** [comment]
We suspect this is a storybook issue and reduced the severity to 3. Will investigate!

**Comment by andy-blum** [comment]
Per slack discussion with @Gopi916 :

> '1.4.12: Text spacing' is an Accessibility guideline which allow a user to override text spacing via user stylesheet, bookmarklet, extension, or application.
> 
> Detailed description on the guideline is available in the below links:
https://www.ibm.com/able/requirements/requirements
https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html
https://www.w3.org/WAI/WCAG21/Techniques/css/C36
You can replicate the issue with text spacing bookmarklet. You can find the bookmarklet in 'Procedure' section in the page 'https://www.w3.org/WAI/WCAG21/Techniques/css/C36'
> 
> [For testing] please use the link https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components/iframe.html?id=components-cta-section--with-content-items

**Comment by andy-blum** [comment]
I'm unable to replicate this issue. @Gopi916 can you please retest and provide screenshots if the issue is still there?

![carbon-design-system github io_carbon-for-ibm-dotcom_next_web-components_iframe html_id=components-cta-section--with-content-items](https://user-images.githubusercontent.com/25532785/183100819-ffc1028e-cd4a-4837-a27e-c70038600588.png)

![carbon-design-system github io_carbon-for-ibm-dotcom_next_web-components_iframe html_id=components-cta-section--with-content-items (1)](https://user-images.githubusercontent.com/25532785/183100895-4dfdd0bf-c41d-46d8-8674-8c0b059584fc.png)

**Comment by proeung** [comment]
Closing this issue out as the bug mentioned is not reproducible. @Gopi916 Feel free to reopen if this issue still persists.

**Comment by gopinathan-n** [comment]
@proeung I tried the same. It is repro'ing for me. Below are the details:

URL: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components/iframe.html?id=components-cta-section--with-content-items&knob-Heading%20(required)_CTASection=Optional%20title%20heading-05%20color%20text-01&knob-Copy%20text%20(optional)_CTASection=Optional%20text%20heading-03%20color%20text-01,%20Lorem%20ipsum%20dolor%20sit%20amet,%20consecteture%20adipiscing%20elit%20sed%20dose.&knob-Add%20Copy_CTASection=true&knob-Add%20CTA_CTASection=true&knob-CTA%20Block%20border_CTASection=true

Firefox Version: 104.0.2

Screenshot:
![image](https://user-images.githubusercontent.com/108443049/190971478-701f9b77-7507-4612-bae6-6c1fe01a93a4.png)

**Comment by andy-blum** [comment]
I'm still unable to reproduce this - perhaps we should go over this one together with @Gopi916,  @kennylam?

**Comment by proeung** [comment]
@kennylam I'm going to assign this issue back to your team's queue as we're unable to reproduce this issue mentioned by @Gopi916.
